apps/condo/domains/ticket/components/TicketId/TicketResidentFeatures.tsx

```diff
@@ -33,76 +34,33 @@ interface PaymentsAvailableIndicatorProps {
     propertyAddress: string
 }
 
-/**
- Availability is determined by:
- 1. If receipts are found at this address for the last 2 months
- 2. Or:
-    At least 1 receipt for the current or previous period has been uploaded to the organization.
-    If the number of receipts for the previous period is not equal to 0,
-    then the number of receipts for the current period must be greater than or equal to the number for the previous one.
- */
-const getIsPaymentsInMobileAppAvailable = (receiptsByProperty, receiptsInCurrentPeriod, receiptsInPreviousPeriod) => {
-    if (receiptsByProperty) return true
-
-    if (receiptsInCurrentPeriod > 0 || receiptsInPreviousPeriod > 0) {
-        if (receiptsInPreviousPeriod === 0) return true
-
-        if (receiptsInCurrentPeriod >= receiptsInPreviousPeriod) {
-            return true
-        }
-    }
-
-    return false
-}
-
-const TWO_MONTHS_AGO = dayjs().startOf('day').subtract(2, 'months').toISOString()
+const LAST_MONTH_BEGINNING = dayjs().subtract(1, 'months').startOf('month').format('YYYY-MM-DD')
```

Comment:
You can shorten 'months' to singular it to make it more semantically
```suggestion
const LAST_MONTH_BEGINNING = dayjs().subtract(1, 'month').startOf('month').format('YYYY-MM-DD')
```